src/isIsogram.test.js

```diff
@@ -6,4 +6,24 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it('no repeating letter', () => {
+    expect(isIsogram('look')).toEqual(false);
+  });
+
+  it('no repeating letter', () => {
+    expect(isIsogram('Peter')).toEqual(false);
+  });
+
+  it('no repeating letter', () => {
+    expect(isIsogram('Adam')).toEqual(false);
+  });
+
+  it('no repeating letter', () => {
+    expect(isIsogram('playgrounds')).toEqual(true);
+  });
+
+  it('no repeating letter', () => {
+    expect(isIsogram('')).toEqual(true);
+  });
```

Comment:
Each test case should have a unique description. This will help you and others understand what each test case is checking. For example, instead of 'no repeating letter' for all, you can describe what exactly this test is doing, like 'should return false for word with repeating letters', 'should return true for word without repeating letters', 'should return true for empty string', etc.